build/changelog.py

```diff
@@ -53,23 +89,30 @@ def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument(
         "--repo_url", default="https://github.com/open-policy-agent/opa")
+    parser.add_argument("token", nargs="?", 
+                        default="", help="GitHub API token")
     parser.add_argument("from_version", nargs="?",
                         default=get_latest_tag(), help="start of changes")
     parser.add_argument("to_commit", nargs="?",
                         default="HEAD", help="end of changes")
     return parser.parse_args()
 
 
+maintainers = get_maintainers()
```

Comment:
NIt: I'd probably move this up to be right after the get_maintainers definition unless there's a reason why it's here.

Reply:
I just thought that it would be more readable if it was near the main function, I have moved it so that it is right after the function definition.